From a15dc7aa607394921397e0f2fa533077042fbda1 Mon Sep 17 00:00:00 2001
From: AKosmachyov <a.kosmachyov@gmail.com>
Date: Wed, 30 Aug 2023 15:49:54 +0300
Subject: [PATCH] Patch ZeroMQ

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e92141d..0187791 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1344,7 +1344,7 @@ if(MSVC)
 else()
   # avoid building everything twice for shared + static only on *nix, as Windows needs different preprocessor defines in
   # static builds
-  if(NOT MINGW)
+  if(NOT MINGW AND IOS)
     add_library(objects OBJECT ${sources})
     set_property(TARGET objects PROPERTY POSITION_INDEPENDENT_CODE ON)
     target_include_directories(
@@ -1378,7 +1378,7 @@ else()
   endif()
 
   if(BUILD_STATIC)
-    if(MINGW)
+    if(MINGW OR IOS)
       add_library(libzmq-static STATIC ${sources} ${public_headers} ${html-docs} ${readme-docs} ${zmq-pkgconfig}
                                        ${CMAKE_CURRENT_BINARY_DIR}/version.rc)
     else()
-- 
2.39.2 (Apple Git-143)

